aptos-move/framework/aptos-framework/sources/fungible_asset.move

```diff
@@ -898,7 +899,7 @@ module aptos_framework::fungible_asset {
         amount: u64
     ) acquires FungibleStore, Supply, ConcurrentSupply, ConcurrentFungibleBalance {
         // ref metadata match is checked in burn() call
-        burn(ref, withdraw_internal(object::object_address(&store), amount));
+        burn(ref, raw_withdraw_internal(object::object_address(&store), amount));
     }
 
     public(friend) fun address_burn_from(
```

Comment:
I wanted to this irrespective, but since you are making it even more different - let's rename this to `address_burn_from_for_gas` so it's clear this shouldn't be used anywhere else